Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/revert.py

```diff
@@ -291,7 +291,7 @@ def main(cls, args, repository, **kwargs):
         if cls.relate_issues(args, repository, issue, commit_issues, revert_reason):
             return 1
 
-        if args.safe is not None and args.pr:
```

Comment:
Should probably be `and args.pr is not False` right? If the user says `--no-pr` and `--safe`, we can't really respect both.

Reply:
Good point! I added a prompt for the user to decide which to follow.